Controllers/ToppingsController.js

```diff
@@ -5,9 +5,34 @@ const ObjectId = require("mongodb").ObjectId;
 
 const getAllToppings = (req, res) => {
   DbService.connectToDb(async (db) => {
-    const toppings = await ToppingsService.getAllToppings(db);
-    res.json(toppings);
+    const result = await ToppingsService.getAllToppings(db);
+    let jsonRes = ResponsesService.successful()
```

Comment:
Make sure you're consistent with your semi colons - either use them everywhere or not at all